src/main/drivers/max7456_symbols.h

```diff
@@ -34,49 +29,49 @@
 // Degrees Icon for HEADING/DIRECTION HOME
 #define SYM_DEGREES 0xBD
 
-// Direction arrows
-#define SYM_ARROW_UP    0x60
+// Direction arrows 
+#define SYM_ARROW_SOUTH 0x60
```

Comment:
Not sure why you changed UP DOWN LEFT RIGHT to NORTH SOUTH WEST EAST. The change is unconventional and confuses rather than clarifies.